### DISCUSSION TITLE
SDAI app is temporary unavailable on Google Play

### DISCUSSION THREAD

**Discussion by ShiftHackZ** [ShiftHackZ/Stable-Diffusion-KMP#350] [Announcements]
Since 05.11.2024 app is temporary unlisted from Google Play because of compliance issues. Our team already submitted an update for the Google Play team review, waiting for their manual review, it may take up to 7 days to get the app listed back. For now use GitHub releases to install any app flavor.

**Comment by ali0une** [comment]
Maybe you could update SDAi on f-droid?

**Comment by ShiftHackZ** [comment]
App is available on Google Play, so closing this announcement.
